src/decorators/exclude.decorator.ts

```diff
@@ -16,8 +16,9 @@ export function Exclude(options: ExcludeOptions = {}): PropertyDecorator & Class
    * decorator only receives one parameter.
    */
   return function (object: any, propertyName?: string | Symbol): void {
-    defaultMetadataStorage.addExcludeMetadata({
+    defaultMetadataStorage.setMetaData({
       target: object instanceof Function ? object : object.constructor,
```

Comment:
`object instanceof ClassConstructor`?

Reply:
I don't understand this. `ClassConstructor` is an interface we cannot use `instanceof` on it.

Reply:
Ops, my mistake.